ipay88/build.gradle

```diff
@@ -38,9 +39,15 @@ android {
     }
 }
 
+tasks.configureEach { task ->
+    if (task.name == "generateReleaseBuildConfig") {
```

Comment:
We have to disable generation of new `BuildConfig` file as gradle does not allow duplicates.